cinder/volume/drivers/dell_emc/unity/adapter.py

```diff
@@ -668,6 +683,16 @@ def _thin_clone(self, vol_params, src_snap, src_lun=None):
                 'thin clone api. source snap: %(src_snap)s, lun: %(src_lun)s.',
                 {'src_snap': src_snap.name,
                  'src_lun': 'Unknown' if src_lun is None else src_lun.name})
+        except storops_ex.UnityThinCloneNotAllowedError:
```

Comment:
Will this be a problem for Mitaka version? I cherried-pick thick volume support first then you cherried-pick thin clone support. I am wondering whether this part of codes existing with your thin clone change.

Reply:
You are right, the thick volume does not work in Mitaka, I will make a patch to fix it.

Reply:
Lots of codes need to be changed when Thin clone back ported, because the previous thick volume back port is base on non-thin-clone codes, it is easy to introduce other issues if I fix this issue manually, so my plan is to cherry-pick this commit from Pike to Mitaka.

Reply:
Agree. But do the same tests on M and P to make sure both functionalities work.